tools/acorn-optimizer.js

```diff
@@ -370,12 +370,20 @@ function runJSDCE(ast, aggressive) {
         ensureData(scopes[scopes.length - 1], node.id.name).def = 1;
       }
       const scope = {};
+      scopes.push(scope);
       node.params.forEach((param) => {
+        if (param.type === 'RestElement') {
```

Comment:
RestElement is `f(a, b, ...c)` .. here `c` is `RestElement`.  I didn't write a specific test for it, but we do have at least one library function with this pattern because I hit the assertion without it.

Do you want me to add a specific test for this?

Reply:
I see, if the assert was already hit then that sounds enough then.